sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/text/TextIOIT.java

```diff
@@ -136,7 +136,7 @@ public void writeThenReadAll() {
 
     PCollection<String> consolidatedHashcode =
         testFilenames
-            .apply("Match all files", FileIO.matchAll())
+            .apply("Match all files", FileIO.matchAll().withOutputParallelization(true))
```

Comment:
We don't need to change this after default set to true

Reply:
setting this to `false`, the idea here is to check the performance test to see if theres a meaningful improvement with skipping the reshuffle vs not.